.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+#### BEFORE YOU SUBIMIT please read the following:
+If you have a support request or question please 
+submit them to [StackOverflow](http://stackoverflow.com/questions/tagged/webpack) using the tag `[webpack]` or the [webpack Gitter](https://gitter.im/webpack/webpack). Future support requests will be closed.
+
+
+
+I'm submitting a...
+=====
+- [ ] bug report
+- [ ] feature request
+- [ ] support request => Please do not submit support request here, see note at the top of this template. 
+
+
+** Webpack version: **
+1.10.x/2.x
+
+** Please tell us about your environment: **
+OSX 10.x / Linux / Windows 10
```

Comment:
I think this lets people "check" the box themself. I can't remember the markdown for the checkbox.